src/app/WriteHandler.cpp

```diff
@@ -63,7 +63,7 @@ void WriteHandler::ClearExistingExchangeContext()
     }
 }
 
-CHIP_ERROR WriteHandler::OnWriteRequest(Messaging::ExchangeContext * apExchangeContext, System::PacketBufferHandle aPayload)
+CHIP_ERROR WriteHandler::OnWriteRequest(Messaging::ExchangeContext * apExchangeContext, System::PacketBufferHandle && aPayload)
```

Comment:
This appears to be not a typo fix, but a logic change?

Reply:
Our general convention now seems to be to pass packetbuffers by rvalue reference, I intend to use rvalue, but i make a typo here which use copy constructor, functionality is same, copy constructor make additional memory copy. Here I correct it to use rvalue reference.